salt/metalk8s/orchestrate/upgrade/etcd.sls

```diff
@@ -0,0 +1,57 @@
+{%- set dest_version = pillar.orchestrate.dest_version %}
+{%- set etcd_nodes = salt.metalk8s.minions_by_role('etcd') %}
+{%- set control_plane_ip = salt.saltutil.runner(
+    'mine.get',
+    tgt=pillar.bootstrap_id,
+    fun='control_plane_ip'
+)[pillar.bootstrap_id] %}
+
+{%- set pillar_data = {
+    'repo': {
```

Comment:
These values are now available in the existing pillar, could you copy them from there? Given that change and the potential improvement mentioned below, it'd remove the need to pass `bootstrap_id` to invoke this.

Reply:
Ping @TeddyAndrieux?

Reply:
I will remove then since #896 done